ani-cli

```diff
@@ -591,7 +591,7 @@ if [ -z "$select_first" ]; then
 			ep_choice_end=
 			;;
 		b)
-			prompt "Select quality"
+			prompt "Select quality. Options (best|worst|360|480|720|1080)"
```

Comment:
For sake of consistency, I'd prefer `"Select quality. (opetions: best|worst|360|480|720|1080)"`